packages/app/src/elements/item-view.ts

```diff
@@ -601,16 +712,28 @@ export class ItemView extends Routing(StateMixin(LitElement)) {
         }
     }
 
-    save() {
+    async save(skipExpiresPrompt = false) {
         if (!this._nameInput.reportValidity()) {
             return;
         }
+
+        if (!skipExpiresPrompt && this._item!.expiresAt) {
```

Comment:
I need this param to not ask this question when setting a new date. The alternative would be to store any "input" value in a state, which seemed more complex and unnecessary.